### DISCUSSION TITLE
python -m spacy download en_core_web_sm failed due to SSL cert error

### DISCUSSION THREAD

**Discussion by avijitghosh77** [explosion/spaCy#13102] [Help: Coding & Implementations]
ERROR: Could not install packages due to an OSError: HTTPSConnectionPool(host='github.com', port=443): Max retries exceeded with url: /explosion/spacy-models/releases/download/en_core_web_sm-3.7.0/en_core_web_sm-3.7.0-py3-none-any.whl (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:997)')))
  WARNING: Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:997)'))': /explosion/spacy-models/releases/download/en_core_web_sm-3.7.0/en_core_web_sm-3.7.0-py3-none-any.whl

**Comment by rmitsch** [comment]
Hi @avijitghosh77, I'm unable to reproduce this. Please provide more information on the system which you're trying to install `en_core_web_sm` on.

**Reply by avijitghosh77** [reply]
I am on dell laptop running windows 10, intel core i5 8th Gen and using Jupyter notebook

**Reply by avijitghosh77** [reply]
I Jupyter Notebook I get this error when I run the code

# Load the English language model from spacy
import spacy
nlp = spacy.load('en_core_web_sm')

Error below
---------------------------------------------------------------------------
OSError                                   Traceback (most recent call last)
Cell In[5], line 3
      1 # Load the English language model from spacy
      2 import spacy
----> 3 nlp = spacy.load('en_core_web_sm')

File ~\AppData\Local\anaconda3\lib\site-packages\spacy\__init__.py:51, in load(name, vocab, disable, enable, exclude, config)
     27 def load(
     28     name: Union[str, Path],
     29     *,
   (...)
     34     config: Union[Dict[str, Any], Config] = util.SimpleFrozenDict(),
     35 ) -> Language:
     36     """Load a spaCy model from an installed package or a local path.
     37 
     38     name (str): Package name or model path.
   (...)
     49     RETURNS (Language): The loaded nlp object.
     50     """
---> 51     return util.load_model(
     52         name,
     53         vocab=vocab,
     54         disable=disable,
     55         enable=enable,
     56         exclude=exclude,
     57         config=config,
     58     )

File ~\AppData\Local\anaconda3\lib\site-packages\spacy\util.py:472, in load_model(name, vocab, disable, enable, exclude, config)
    470 if name in OLD_MODEL_SHORTCUTS:
    471     raise IOError(Errors.E941.format(name=name, full=OLD_MODEL_SHORTCUTS[name]))  # type: ignore[index]
--> 472 raise IOError(Errors.E050.format(name=name))

OSError: [E050] Can't find model 'en_core_web_sm'. It doesn't seem to be a Python package or a valid path to a data directory.

**Reply by rmitsch** [reply]
Looks like you haven't installed `en_core_web_sm`. In Jupyter this should work with `!python3 -m spacy download en_core_web_sm`.
